bots/iam_user_detach.py

```diff
@@ -0,0 +1,52 @@
+'''
+## iam_user_detach
+Log.ic bot only
+What it does: Detaches an IAM user from an IAM group.
+Usage: AUTO: iam_user_detach
+Limitations: The bot will stop running if the proper 'AddUserToGroup' event is not found
+             The bot will not notify if the IAM user is already detached or was not attached to the group in the
+                 first place.
+'''
+
+import boto3
+import botocore.exceptions
+import json
+import bots_utils
+
+
+EVENT_NAME = 'AddUserToGroup'
+
+
+def run_action(boto_session, rule, entity, params):
+    text_output = ''
+    user_to_remove = entity.get('name')
+
+    # Get event from cloud trail
+    event = bots_utils.cloudtrail_event_lookup(boto_session, entity, EVENT_NAME,
+                                               resource_name_to_filter=user_to_remove)
+    if not event:
+        # If valid event was not found - bot will not run
+        text_output = f'Error: No matching \'{EVENT_NAME}\' events were found in cloud trail. Bot wasn\'t executed.'

```

Comment:
No need to use those \, it's more elegant to use {}

Reply:
Did you mean -  \'   ?

Reply:
I meant - \